config/ember-try.js

```diff
@@ -7,23 +7,25 @@ module.exports = async function() {
     useYarn: true,
     scenarios: [
       {
-        name: 'ember-3.17',
+        name: 'ember-lts-3.20',
```

Comment:
**question**

In `v4.8`, [Ember's blueprint](https://github.com/ember-cli/ember-cli/blob/v4.8.0/blueprints/addon/files/addon-config/ember-try.js) lists `ember-lts-3.28` and `ember-lts-4.4` because they are the last 2 LTS versions.

I wondered if there is a reason to target `3.20` and skip `3.24` and `3.28`. (Maybe to cover a broader range without testing every LTS in-between?)

Reply:
Ah, I see. I didn't check the existing content of `travis.yml`.

I'd say the versions of Ember that you want to check in CI (i.e. which versions do you want to claim to support in the `README` file) are up to you (and maybe other maintainers-to-be). Maybe ask yourself: How do I plan to update the addon?

What I mean by above is, if you list `3.28` and `4.4` because they are the latest 2 LTS versions, there's less guarantee that the addon, after you update the underlying code, will continue to work in apps that use an older version of Ember. On the other hand, supporting the latest 2 LTS versions might give you more freedom to how you want to update the code (e.g. use a latest feature in Ember).

Alternatively, if you list `3.20` and `4.4`, then you might need to declare a (second or third) major release when you later drop support for older LTS versions.

I personally would not list `3.17` in CI (not an LTS).

Reply:
> I'd say the versions of Ember that you want to check in CI (i.e. which versions do you want to claim to support in the README file) are up to you (and maybe other maintainers-to-be). Maybe ask yourself: How do I plan to update the addon?

Thanks for the advice, really appreciate it! I think for now I will keep the versions as is so we can update some of the other dependencies, namely the actual Launchdarkly client SDK. This way we can still let people have a most up to date v2. After this I'll look into doing another major release supporting newer versions of ember.

Really appreciate all the feedback!